core/domain/services/graphql/connection_resolvers/TicketConnectionResolver.php

```diff
@@ -113,6 +116,28 @@ public function get_query_args()
             );
         }
 
+        // If default tickets should be included.
+        if (! empty($this->args['where']['includeDefaultTickets'])) {
+            /**
+             * We need to get each ticket that
+             * - satisfies $where_params above
+             * OR
+             * - it's a default ticket
+             */
+            $where_params = [
+                'OR' => [
+                    // use extra OR instead of AND to avoid it getting overridden
+                    'OR' => [
+                        'AND' => [
+                            'TKT_deleted'    => ['IN', [true, false]],
+                            'TKT_is_default' => 1,
+                        ]
+                    ],
+                    'AND' => $where_params,
+                ],
```

Comment:
how long did it take you to wrap your head around this?
i find some of this logic in the models to be very UN-intuitive...
so good job figuring this out

Reply:
Yeah, it definitely IS un-intuitive but we have to deal with it 😄